$Id$
ash does not understand the test '-a' (test for bare existence)
--- ltp-full-20061121.orig/runltp	2006-11-21 21:31:20.000000000 +0100
+++ ltp-full-20061121/runltp	2006-12-06 13:22:14.000000000 +0100
@@ -285,7 +285,7 @@ main()
                      ${LTPROOT}/runtest/sched ${LTPROOT}/runtest/math \
                      ${LTPROOT}/runtest/nptl ${LTPROOT}/runtest/pty
         do
-            [ -a "$SCENFILES" ] || \
+            [ -e "$SCENFILES" ] || \
             {
                 echo "FATAL: missing scenario file $SCENFILES"
                 exit 1
@@ -319,7 +319,7 @@ main()
                          ${LTPROOT}/runtest/rpc \
                          ${LTPROOT}/runtest/nfs
         do
-            [ -a "$SCENFILES" ] || \
+            [ -e "$SCENFILES" ] || \
             { 
                 echo "FATAL: missing scenario file $SCENFILES"
                 exit 1
